Recruit-Training/Basic-Recruit-Training/person/main.cpp

```diff
@@ -17,17 +17,31 @@ void compareAnswers(const std::string myAnswer, const std::string correctAnswer)
         std::cout << "Correct." << std::endl;
     }
 }
+
+void myIncrement(int* x);
+void mySwap(int& a, int& b);
+
 // Swaps a & b.
 void mySwap(int& a, int& b)
```

Comment:
This function **uses references** for its arguments.
So what you're doing here with pointers is unnecessary.

Try this function out without pointers!